### ISSUE TITLE
Research meeting WCAG Standards

### ISSUE THREAD

**Issue by acoard-aot** [bcgov/foi-flow#224]
Title of ticket: 

#### Description
Ensure we have regulatory compliance with accessibility.

Summary: https://www2.gov.bc.ca/gov/content/governments/about-the-bc-government/accessibility/legislation/summary
Full text: https://www.leg.bc.ca/parliamentary-business/legislation-debates-proceedings/42nd-parliament/2nd-session/bills/first-reading/gov06-1

When we have tasks they will be under a WCAG Epic

#### DOD
- [x] Research11y legislative requirements from new act
- [ ] Determine/Run screen reader to see how we are meeting the WCAG Standard
- [ ] Create tasks to meet WCAG Standard

(does NOT include actual dev work, just reports for future tasks).

**Comment by lmullane** [comment]
@acoard-aot, we should revisit this ticket to ensure we don't have too much work later.

**Comment by mpilchar** [comment]
@acoard-aot @JHarrietha-AOT Is this the same as the accessibility plug-in we discussed during our Release planning/identifying risks? If so, we discussed having it as part of this sprint but may have forgotten to bring it in during sprint planning. Let's discuss, and determine if there is work that should be reprioritized

**Comment by lmullane** [comment]
@acoard-aot, @JHarrietha-AOT, can we close this ticket?

I believe we tested against WCAG Standards, correct?

**Comment by acoard-aot** [comment]
Yes, we can close this ticket.  I'll mark as closed with this comment.
